src/com/esotericsoftware/kryo/serializers/FieldSerializerConfig.java

```diff
@@ -136,4 +138,12 @@ public boolean isCopyTransient() {
     public boolean isSerializeTransient() {
         return serializeTransient;
     }
+
+    public FieldSerializer.CachedFieldNameStrategy getCachedFieldNameStrategy() {
+        return cachedFieldNameStrategy;
+    }
+
+    public void setCachedFieldNameStrategy(FieldSerializer.CachedFieldNameStrategy cachedFieldNameStrategy) {
+        this.cachedFieldNameStrategy = cachedFieldNameStrategy;
```

Comment:
Add `if (TRACE) trace("kryo.FieldSerializerConfig", "CachedFieldNameStrategy: " + cachedFieldNameStrategy);` as in other methods.